app/renderer/abr-document.js

```diff
@@ -350,6 +366,65 @@ AbrDocument.prototype = {
         return this.save(path, callback);
     },
 
+    initWatcher: function () {
+        var that = this;
+        var updateTitle = function () {
```

Comment:
Why don't you simply use `abrDoc.updateWindowTitle()` here instead of creating a duplicate of it?